include/deal.II/dofs/dof_accessor.h

```diff
@@ -2039,6 +2039,46 @@ class DoFCellAccessor : public DoFAccessor<dimension_,
    */
   void
   clear_future_fe_index() const;
+
+  /**
+   * Return the index of the finite element that is dominated by those assigned
+   * as future finite elements to the children of this cell.
```

Comment:
"the index" here is the one among the ones you are finding on the child cells, right, not among all possible indices? That's because your second paragraph seems to suggest that you're only considering all other indices in the second step. Maybe clarify that here.

Reply:
I'll add another sentence for the second paragraph.